lib/generators/react_on_rails/base_generator.rb

```diff
@@ -18,13 +18,21 @@ class BaseGenerator < Rails::Generators::Base
                    desc: "Install Redux gems and Redux version of Hello World Example",
                    aliases: "-R"
 
+      def change_webpacker_source_path
+        webpacker_yml = File.join(destination_root, "config/webpacker.yml")
+        return unless File.exist?(webpacker_yml)
+
+        contents = File.read(webpacker_yml).gsub(/source_path:.*/, "source_path: client/app")
```

Comment:
Let's think about this.

* Probably, we should match exactly.
* The problem case is when this was configured differently.
* Webpacker may have been previously used for Vue.js, or even React
* We will be changing the directory to what we think is the best approach
* rails/webpacker and the app will break b/c this is changed
* We can do one of 2 things:
  * Not change the default in this case
  * Add such details to the post-generation message so that it's easy for the developer to revert this change.

Reply:
Could we DRY up `client/app` so that this case and other places use this?

Reply:
Let's update this file: https://github.com/shakacode/react_on_rails/blob/master/lib/generators/USAGE

to indicate where the files are installed and what you should do to revert back to the rails app/javascript default.

Indicate:

```
The team at ShakaCode prefers to have all client side development files in a top-level directory called /client. This more cleanly separates the front-end code from the back-end code.

To change back to the default...
```

try to finish up this message to what you think is best.

Reply:
Make sure this message runs after running the generator.

Reply:
@wanglixiang1992 Let's use a constant for "client/app" to DRY this up. DRY = Don't Repeat Yourself